e2e/modal/modal.e2e-spec.ts

```diff
@@ -75,4 +92,71 @@ test.describe.parallel(`Modal tests`, () => {
 		await modalPO.locatorCloseButton.click();
 		await modalPO.locatorRoot.waitFor({state: 'hidden'});
 	});
+
+	test(`Modal customizability`, async ({page}) => {
+		const modalPO = new ModalPO(page, 0);
+		const expectedState: State = {
+			rootClasses: ['d-block', 'fade', 'modal', 'show'],
+			header: undefined,
+			title: undefined,
+			body: 'Body',
+			footer: undefined,
+			closeButton: undefined,
+		};
+
+		await test.step(`Modal with only body`, async () => {
```

Comment:
Instead of doing steps, I thing it would be better to have a normal flow with a `describe` and separated `test`. The point here is that each test are dependant on the previous one. At the end, it can be difficult to quickly understand how is built the expected states through the different steps. 

Also, maybe it would be a good idea to have a function that returns the expectedState, so that a new object will be created each time `getExpectedState` will be called, to ensure to work with an independant object all the time.